beacon-chain/blockchain/head.go

```diff
@@ -143,6 +144,12 @@ func (s *Service) saveHead(ctx context.Context, headRoot [32]byte) error {
 			},
 		})
 
+		if featureconfig.Get().CorrectlyInsertOrphanedAtts {
```

Comment:
Let's move this inside of the saveOrphanedAtts function so that callers dont need to keep track of feature flags